### PR TITLE
Make `pihole -up` a bit more intelligent

### PR DESCRIPTION
**Pull request by PromoFaux**
Fixes #[internal] .

Changes proposed in this pull request:

- Runs version check before updating files and makes the output a bit clearer

- points user at the changelog at the end

- tells user if nothing has been updated

@pi-hole/gravity

### CODE REVIEW

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -543,11 +543,10 @@ installScripts() {
 	${SUDO} cp /etc/.pihole/advanced/Scripts/blacklist.sh /opt/pihole/blacklist.sh
```
Comment (dschaper) [inline automated install/basic-install.sh]: This all probably should be redone with `install` instead, I think it would flow better.

**Comment by dschaper** [comment]
:+1:

**Comment by dschaper** [comment]
:+1:

**Comment by PromoFaux** [comment]
:shipit:

**Comment by jacobsalmela** [comment]
:shipit:

**Pull request merged**